apps/scan/backend/src/electrical_testing/app.ts

```diff
@@ -105,6 +109,41 @@ function buildApi({
       }
     },
 
+    async getLatestScannedSheet(): Promise<SheetOf<string> | null> {
+      const allScannedImageNames = await readdir(workspace.ballotImagesPath);
+      const latestElectricalTestingImageName = await iter(allScannedImageNames)
+        .async()
+        .filter((name) => name.startsWith('electrical-testing-'))
+        .maxBy(
+          async (name) =>
+            (await stat(join(workspace.ballotImagesPath, name))).mtimeMs
```

Comment:
Instead of having to stat all the files in the directory, could we store the files with a timestamp in their file name and then use an alphabetical sort on the names?

Probably not a huge deal since stat isn't a particularly expensive call